app/src/main/java/com/sunasterisk/a14day_challenge/data/local/DataBaseHandler.kt

```diff
@@ -97,30 +95,38 @@ class DataBaseHandler(context: Context): SQLiteOpenHelper(context, DATABASE_NAME
         cursor.moveToFirst()
 
         while (!cursor.isAfterLast) {
-            val user = User(
-                cursor.getString(0),
-                cursor.getString(1),
-                cursor.getString(2),
-                cursor.getString(3),
-                cursor.getDouble(4),
-                cursor.getDouble(5),
-                cursor.getInt(6)
-            )
-
+            val user = User(cursor)
             userList.add(user)
             cursor.moveToNext()
         }
         cursor.close()
         return userList
     }
 
-    companion object{
+    fun addUser(user: User) {
+        var db = this.writableDatabase
+
+        val values = ContentValues().apply {
+            put(USER_ACC, user.account)
+            put(USER_NAME, user.name)
+            put(USER_PASS, user.password)
+            put(USER_BIR, user.birthday)
+            put(USER_HEIGHT, user.height)
+            put(USER_WEIGHT, user.weight)
+            put(USER_PROCESS, user.process)
+        }
+        db.insert(TABLE_USER, null, values)
+        db.close()
```

Comment:
Có thể tạo hàm getContentValues trong model User 
```suggestion

        writableDatabase.run {
            insert(TABLE_USER, null, user.getContentValues())
           close()
       }
```